src/types/index.ts

```diff
@@ -338,6 +338,7 @@ export type PaymentDetails = {
   totalAmount: string;
   assetCode: string;
   statusHistory: PaymentDetailsStatusHistoryItem[];
+  externalPaymentId: string;
```

Comment:
Should we make it optional?

```suggestion
  externalPaymentId?: string;
```

Reply:
oooh yes!